source/expressions.tex

```diff
@@ -4319,7 +4319,7 @@
 The address of an overloaded function\iref{over} can be taken
 only in a context that uniquely determines which version of the
 overloaded function is referred to (see~\ref{over.over}).
-Since the context might determine whether the operand is a static or
+Since the context can determine whether the operand is a static or
```

Comment:
```suggestion
Since the context can affect whether the operand is a static or
```
... would better parallel the rest of the sentence, and avoids suggesting the context is the only factor that influences the decision in some cases.

Reply:
Great!